### ISSUE TITLE
Can't import cjs

### ISSUE THREAD

**Issue by glassdimlygr** [axios/axios#5358]
### Describe the bug

Docs say you can import cjs with `const axios = require('axios/dist/browser/axios.cjs');`

This result in:

```
[dev] internal/modules/cjs/loader.js:430
[dev]       throw e;
[dev]       ^
[dev]
[dev] Error [ERR_PACKAGE_PATH_NOT_EXPORTED]: Package subpath './dist/browser/axios.cjs' is not defined by "exports" in /Users/jjohn/www/grate-cms/node_modules/axios/package.json
[dev]     at throwExportsNotFound (internal/modules/esm/resolve.js:299:9)
[dev]     at packageExportsResolve (internal/modules/esm/resolve.js:522:3)
[dev]     at resolveExports (internal/modules/cjs/loader.js:424:36)
[dev]     at Function.Module._findPath (internal/modules/cjs/loader.js:464:31)
[dev]     at Function.Module._resolveFilename (internal/modules/cjs/loader.js:802:27)
[dev]     at Function.Module._load (internal/modules/cjs/loader.js:667:27)
[dev]     at Module.require (internal/modules/cjs/loader.js:887:19)
[dev]     at require (internal/modules/cjs/helpers.js:74:18)
[dev]     at Object.<anonymous> (/Users/jjohn/www/grate-cms/src/server/env.js:4:15)
[dev]     at Module._compile (internal/modules/cjs/loader.js:999:30) {
[dev]   code: 'ERR_PACKAGE_PATH_NOT_EXPORTED'
[dev] }

```

### To Reproduce

`const axios = require('axios/dist/browser/axios.cjs');`

### Code snippet

```js
`const axios = require('axios/dist/browser/axios.cjs');`
```

### Expected behavior

No error, works.

### Axios Version

1.1.3

### Browser Version

n/a

### Node.js Version

v12.22.12

**Comment by seesemichaelj** [comment]
If I'm understanding the [NodeJS docs for exports and conditional exports](https://nodejs.dev/en/api/v18/packages/) properly, `require("axios/dist/node/axios.cjs")` should not work at all. For it to work, `package.json` would need to have:

``` json
{
  "exports": {
    "./dist/node/axios.cjs": "./dist/node/axios.cjs"
  }
}
```

As it currently stands, the only supported require paths are `require("axios")` and `require("axios/package.json")`. The former should point to `./dist/node/axios.cjs` in a non-browser target.

So in other words, you should be able to just use `require("axios")` to get the CJS file. `import "axios"` will get the ES Module file

**Comment by DigitalBrainJS** [comment]
As stated in the Readme, direct loading by path is a workaround for bundlers and older node versions (`<v12.16.0`) that do not yet support conditional export. For modern node environments and tools, you can only use `require('axios')` / `import 'axios'` which maps them to the appropriate entry points. 
Node `v12.22.12` definitely supports conditional export, so that is why you can see the error when trying to load the entry point by its path.

**Comment by Unitech** [comment]
This PR is not working ?

```
node index.js 
node:internal/modules/cjs/loader:553
      throw e;
      ^

Error [ERR_PACKAGE_PATH_NOT_EXPORTED]: Package subpath './dist/node/axios.cjs' is not defined by "exports" in /home/unitech/keymetrics/pm2-runtime/pm2-io-js-api/node_modules/axios/package.json
    at new NodeError (node:internal/errors:405:5)
    at exportsNotFound (node:internal/modules/esm/resolve:260:10)
    at packageExportsResolve (node:internal/modules/esm/resolve:590:9)
    at resolveExports (node:internal/modules/cjs/loader:547:36)
    at Module._findPath (node:internal/modules/cjs/loader:621:31)
    at Module._resolveFilename (node:internal/modules/cjs/loader:1034:27)
    at Module._load (node:internal/modules/cjs/loader:901:27)
    at Module.require (node:internal/modules/cjs/loader:1115:19)
    at require (node:internal/modules/helpers:119:18)
    at Object.<anonymous> (/home/unitech/keymetrics/pm2-runtime/pm2-io-js-api/src/network.js:4:15) {
  code: 'ERR_PACKAGE_PATH_NOT_EXPORTED'
}
```

**Comment by jasonsaayman** [comment]
closing as i belive we have solve this
